buildtest/cli/config.py

```diff
@@ -118,6 +121,11 @@ def validate_config(configuration):
     console.print(f"{configuration.file} is valid")
 
 
+def view_path(configuration):
+    """Display the path to configuration file regardless if file is valid"""
```

Comment:
Can you add the following in docstrings. I think this is mentioned above in several methods

```
    Args:
        configuration (buildtest.config.SiteConfiguration): An instance of SiteConfiguration class
```